### ISSUE TITLE
Make add location more like edit location process

### ISSUE THREAD

**Issue by ezwelty** [falling-fruit/falling-fruit-web#604]
When adding a location, going back from the location form (/locations/new) to the map to set a location (/locations/init), and back to the form results in a blank form. In other words, it isn't possible to adjust the location position without restarting the whole add-location process.

I propose that it should be possible to adjust the position of a new location without losing form content. I suggest that this can be accomplished by treating the add-location form in a similar same way as the edit-location form, such that the location position is orange and links to the edit-position page, and (unlike edit-location) clicking on back does the same.

**Comment by wbazant** [comment]
I agree with the crux of the issue - allowing the adjustment of the position - and there would be other usability benefits to also making it possible to go back -> settings -> language, or back -> sign in.

Not sure whether also making the edit location orange also for adding location is good or not - it produces consistency between the forms, but offering multiple ways to do the same thing can be confusing.

Implementation note: there's already functionality to save form contents and bring it back, so this shouldn't be too bad to solve!
